pennylane/interfaces/torch.py

```diff
@@ -135,38 +170,47 @@ def forward(ctx, input_kwargs, *input_):
             return torch.from_numpy(res)
 
         @staticmethod
+        @once_differentiable
```

Comment:
Ah I remember we were discussing that here: https://github.com/XanaduAI/pennylane/pull/612
And in the end we decided not to use it. Does it work now?

Reply:
It still doesn't work in the functional interface, but it feels weird not to include it, since the PyTorch API recommends it (and uses it themselves!). It could just be a matter of the new high-level functional interface functions `torch.autograd` being relatively new additions 🤔 

Either way, even if it doesn't do anything, it serves as a reminder to developers that this method is only once differentiable 😆